docs/notebooks/downstream_machine_learning_regression_evaluation.ipynb

```diff
@@ -0,0 +1,311 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluate synthetic data on downstream regression models\n",
+    "\n",
+    "### How to use this notebook\n",
+    "Use this notebook to understand the quality and performance of your synthetic or augmented data on downstream machine learning regression tasks. \n",
+    "\n",
```

Comment:
suggestion: similar to classification, mention comparison to real world data here.

Reply:
added